test/record/relationTest.js

```diff
@@ -65,4 +65,29 @@ describe('Viking.Relation', () => {
             });
         })
     })
+    
+    describe('order', () => {
+        it('single key', function () {
+            let relation = Model.where({parent_id: 11}).order('created_at')
+            relation = relation.reorder({created_at: 'desc'})
+            
+            relation.load()
+            assert.ok(this.findRequest('GET', '/models', { params: { where: {parent_id: 11}, order: {created_at: 'desc'} } }));
+        })
+        
+        it('hash', function () {
+            let relation = Model.where({parent_id: 11}).order({created_at: 'asc'})
+            
```

Comment:
no reorder here? remove test or test `{created_at: 'asc'}`?

Reply:
Looks like it's below, maybe delete this one and rename the below one to with hash?